src/ui/components/AnswerOption/AnswerOption.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+import type { IProps } from './types/IProps';
+
+const checkSvg = (
```

Comment:
The SVG is defined as a module-level constant that includes a <defs> with a fixed id ('paint0_linear'). If multiple AnswerOption components render on the same page, duplicate IDs in the DOM will cause unexpected rendering issues and invalidate the document's uniqueness of IDs.

**Sugerencia:**
```typescript
Generate a unique id per component instance or inline the <svg> inside the component and use React.useId() to create a unique id for the gradient. Example:

const id = React.useId();
<svg ...>
  <defs>
    <linearGradient id={`paint0_linear_${id}`} ...>
      ...
    </linearGradient>
  </defs>
  <circle fill={`url(#paint0_linear_${id})`} />
</svg>

Alternatively, avoid <defs> and use a single solid fill or CSS-based SVG coloring to remove id collisions.
```